ECOSYSTEM.md

```diff
@@ -10,4 +10,5 @@ Plugins and tools maintained by the knex team are listed under [Core](#core) whi
 #### [Community](#community)
 - [`knex-paginate`](https://github.com/felixmosh/knex-paginate) Extension of the query builder with `paginate` method that helps with pagination tasks.
 - [`knex-postgis`](https://github.com/jfgodoy/knex-postgis) Extension of Knex with [PostGIS](https://postgis.net/) (geographic objects in PostgreSQL) support.
+- [`knex-serverless-mysql`](https://github.com/MatissJanis/knex-serverless-mysql) Knex dialect for usage with [`serverless-mysql`](https://github.com/jeremydaly/serverless-mysql).
```

Comment:
Could you please add brief explanation what is its purpose?

Reply:
Sure. Added a slightly longer description.